src/panels/githubissues.js

```diff
@@ -0,0 +1,79 @@
+var React = require('react');
+var $ = require('jquery');
+var classNames = require('classnames');
+
+/**
+ * Github Issues panel.
+ */
+var IssuesPanel = React.createClass({
+    getInitialState: function() {
+        return {
+            issues: []
+        };
+    },
+
+    componentDidMount: function() {
+        $.getJSON('https://api.github.com/repos/IllinoisWCS/IllinoisWCS.github.io/issues')
```

Comment:
Why is this request duplicated here and in `updateData`? That means the request would be fired twice on mount.